iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -219,6 +220,8 @@ def _adjust_operator_bundle(
         no custom behavior will be applied.
     :param bool pinned_by_iib: whether or not the bundle image has already been processed by
         IIB to perform image pinning of related images.
+    :param bool recursive_related_bundles: whether or not to terminate customizations when
+        related_bunldes customization is reached.
```

Comment:
`related_bundles`, not `related_bunldes`.

The description and the variable name don't match. Either change the description to be based off of the behavior for recursive related bundles or change the parameter to be based on stopping after related bundles.